bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiBaseThingHandler.java

```diff
@@ -75,59 +80,78 @@ public final void initialize() {
      */
     @SuppressWarnings("null")
     private final @Nullable UniFiController getController() {
-        Bridge bridge = getBridge();
+        final Bridge bridge = getBridge();
         if (bridge != null && bridge.getHandler() != null
                 && (bridge.getHandler() instanceof UniFiControllerThingHandler)) {
             return ((UniFiControllerThingHandler) bridge.getHandler()).getController();
         }
         return null;
     }
 
+    private @Nullable E getEntity() {
+        final UniFiController controller = getController();
+        return controller == null ? null : getEntity(controller);
+    }
+
     @Override
-    public final void handleCommand(ChannelUID channelUID, Command command) {
+    public final void handleCommand(final ChannelUID channelUID, final Command command) {
         logger.debug("Handling command = {} for channel = {}", command, channelUID);
         // mgb: only handle commands if we're ONLINE
         if (getThing().getStatus() == ONLINE) {
-            UniFiController controller = getController();
-            if (controller != null) {
-                E entity = getEntity(controller);
-                if (entity != null) {
-                    if (command == REFRESH) {
-                        refreshChannel(entity, channelUID);
-                    } else {
-                        try {
-                            handleCommand(entity, channelUID, command);
-                        } catch (UniFiException e) {
-                            logger.warn("Unexpected error handling command = {} for channel = {} : {}", command,
-                                    channelUID, e.getMessage());
+            final E entity = getEntity();
+            final UniFiController controller = getController();
+
+            if (entity != null && controller != null) {
+                if (command == REFRESH) {
+                    refreshChannel(entity, channelUID);
+                } else {
+                    try {
+                        if (!handleCommand(controller, entity, channelUID, command)) {
+                            logger.info("Ignoring unsupported command = {} for channel = {}", command, channelUID);
                         }
+                    } catch (final UniFiException e) {
+                        logger.warn("Unexpected error handling command = {} for channel = {} : {}", command, channelUID,
+                                e.getMessage());
                     }
                 }
+            } else {
+                // mgb: set the default state if we're online yet cannot find the respective entity
+                // updateState(channelUID, getDefaultState(channelUID)State);
```

Comment:
I know I'm reviewing a draft PR, but just a reminder about implementing or removing this.